src/cmd-buildextend-metal

```diff
@@ -181,6 +181,11 @@ fi
 
 disk_args=()
 
+sysroot_ro="$(python3 -c 'import sys, yaml; v=yaml.safe_load(sys.stdin).get("sysroot-ro", False); print("1" if v else "")' < "$configdir/image.yaml")"
```

Comment:
Bikeshed, maybe let's just spell it out, e.g. `sysroot-readonly`? That way it also matches the sysroot repo config knob.